testing/mtd_config_fs/Kconfig

```diff
@@ -0,0 +1,49 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config TESTING_FAILSAFE_MTD_CONFIG
```

Comment:
let's change prefix from TESTING_FAILSAFE_MTD_CONFIG_ to TESTING_MTD_CONFIG_FAILSAFE_